sql/sample_data.sql

```diff
@@ -0,0 +1,23 @@
+INSERT INTO rooms (name) VALUES
+	('$Kitchen'),
+	('$Shed');
+
+INSERT INTO users (name, password_hash) VALUES
+	('$Fox', '$2a$10$V5.UzTYmeYh.bPz51WiIH.Yp2KawEqEmgF/amTTXtOHBvcjkFuIrC'),   -- Password: Pa$$w0rd
+    ('$Goat', '$2a$10$Rp2rFH12j0Ovc8VUfJKEX.O2SKHDpHs1b6KBkCqluzSMOowuDagk2'),  -- Password: $s3cr37
+    ('$Cat', '$2a$10$V5.UzTYmeYh.bPz51WiIH.Yp2KawEqEmgF/amTTXtOHBvcjkFuIrC'),   -- Password: Pa$$w0rd
+    ('$Camel', '$2a$10$Rp2rFH12j0Ovc8VUfJKEX.O2SKHDpHs1b6KBkCqluzSMOowuDagk2'); -- Password: $s3cr37
```

Comment:
nit: these lines have 4 space indents, the others have a single tab. I assume you have tabs set to 4 so you do not see this, but it shows up in github.